hub/policy/src/main/java/uk/gov/ida/hub/policy/configuration/InfinispanConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.ida.hub.policy.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.validation.Valid;
+
+public class InfinispanConfiguration {
+
+    @Valid
+    @JsonProperty
+    protected boolean enabled;
+
+    @Valid
+    @JsonProperty
+    private uk.gov.ida.shared.dropwizard.infinispan.config.InfinispanConfiguration configuration;
+
+    public boolean isEnabled() {
```

Comment:
`isEnabled()` method is used in `StateControllerFactoryTest.java` only. Is the method needed in the future?